crates/astria-conductor/src/block_verifier.rs

```diff
@@ -46,22 +50,21 @@ use tracing::{
 /// `validate_signed_namespace_data` is used to validate the data received from the data
 /// availability layer. `validate_sequencer_block` is used to validate the blocks received from
 /// either the data availability layer or the gossip network.
-pub struct BlockVerifier {
-    sequencer_client: Client,
+#[async_trait]
+pub trait BlockVerifier {
```

Comment:
I don't think this should be an extension trait. Just toss the http client in there and import the extension trait. Apart from that nothing in conductor should change.